android/src/main/java/com/reactnativestripesdk/PaymentSheetFragment.kt

```diff
@@ -94,9 +90,9 @@ class PaymentSheetFragment : Fragment() {
                                           addressBundle?.getString("postalCode"),
                                           addressBundle?.getString("state"))
       defaultBillingDetails = PaymentSheet.BillingDetails(address,
-                                                          billingDetailsBundle?.getString("email"),
-                                                          billingDetailsBundle?.getString("name"),
-                                                          billingDetailsBundle?.getString("phone"))
+                                                          billingDetailsBundle.getString("email"),
```

Comment:
nit: I have kinda like the styling of all parameters on a new line, then they also alight closer to the left.